packages/client/src/Components/Modals/DeleteConfirmation.jsx

```diff
@@ -23,26 +23,32 @@ const DeleteModal = () => {
     const { type, data } = currentModal.data
     const curTab = wargame.currentTab
 
```

Comment:
@lilitkarapetyan -  I've switched to a select, so we have a `default` branch.

Do you have any objection to this?